### DISCUSSION TITLE
What is Classes in PimCore?

### DISCUSSION THREAD

**Discussion by darmayantiPuteri** [pimcore/pimcore#8561] [Q&A / Support]
I’m Magento Developer 
Right now I am working on one of my Magento Project that looking for Product Information Management since maintaining a lot of products in Magento can be overwhelming. 
so we decided to use PimCore as the PIM platform. 

We have some questions regarding “Classes" ,
my team is working on Attribute mapping from PimCore to Magento, 
Firstly we are using classes in PimCore as “Attribute Set” in Magento 
Layout Component as “Attribute Group Set” (using TabPanel type) 
Data Component as “Attributes" 

But, after looking back, seems like Classes are NOT Attribute Set.
Attribute sets can be defined as a list of attributes where all the characteristics of a product are demonstrated. For a new product, the attribute set works as a template. Every product must belong to a specific attribute set.

We are wondering, What are classes here ? 
How did you conceptualize the attribute set in PimCore? 

Looking forward your response. 
Thank You for the support.

**Comment by aarongerig** [comment]
Hi @darmayantiPuteri 

Maybe following links are helpful to you?

- https://go.divante.com/magento-pimcore-connector/
- https://github.com/DivanteLtd/pimcore-magento2-bridge

Hope this helps!

**Reply by darmayantiPuteri** [reply]
Hi @aarongerig 

Yes, I have installed both those links.
the connector works as expected, but still not getting the "meaning" or definitions about Classes

**Reply by darmayantiPuteri** [reply]
in the PimCore, what is "Classes" @aarongerig ?

**Reply by passioneight** [reply]
You probably want to have a look at Pimcore's documentation. Specifically, [object classes](https://pimcore.com/docs/pimcore/current/Development_Documentation/Objects/Object_Classes/index.html) .

Maybe the  [classification store](https://pimcore.com/docs/pimcore/current/Development_Documentation/Objects/Object_Classes/Data_Types/Classification_Store.html) and [object bricks](https://pimcore.com/docs/pimcore/current/Development_Documentation/Objects/Object_Classes/Data_Types/Object_Bricks.html) are also of interest to you.
